server/models/Squashes.js

```diff
@@ -2,18 +2,19 @@ var Promise = require('bluebird');
 var Squashes = module.exports;
 
 
-var Squashes =  [ {user: {uid: 1, username: 'john314', avatarUrl: 'http://www.avatarsdb.com/avatars/The_cat.gif'},title: 'I got it!!!', content: 'Turn out I needed to restart my computer!!!', 
-                    comments: [{content: 'dude this is sweet!!' , user: {uid: 2, username: 'eric122', avatarUrl: 'http://www.avatarsdb.com/animals/cats/cat-rain-avatar-52022.htm'}}, {content: 'I really think you are devine and stuff.', user: {uid: 3, username: 'mrKITTTYYYY!', avatarUrl: 'http://www.avatarsdb.com/animals/cats/cat-rain-avatar-52040.htm'}}]},
-                  {user: {uid: 2, username: 'eric122', avatarUrl: 'http://www.avatarsdb.com/animals/cats/cat-rain-avatar-52022.htm'},title: 'I had an epiphany', content: 'I just had to press edit undo, and know everything works again!'},
-                  {user: {uid: 3, username: 'mrKITTTYYYY!', avatarUrl: 'http://www.avatarsdb.com/animals/cats/cat-rain-avatar-52040.htm'},title: 'I really had a bad bug', content: 'BUT MY KITTY SQUASHED IT NOM NOM NOM NOM!'}
-                ];
-module.exports.squashes = function (id) {
-    return  Squashes;
-}
-
+var sampleSquashes =  [ {squashId: 1, user: {uid: 1, username: 'MeyerSauce21', avatarUrl: 'http://www.avatarsdb.com/avatars/The_cat.gif'},title: 'I got it!!!', content: 'Turn out I needed to restart my computer!!!'},
```

Comment:
When working with a record, it's id should always be `id` (if it's something else, that implies a foreign id). I would change `squashId` to `id`